### PR TITLE
Sync internal f7c52af6

### PR DESCRIPTION
**Pull request by liwenchangbdbz**
* [AIT] Disabled tf32 by default, upgraded to 2f61d70c4b14d59f557c1fd9a98cbb5030307f9d
* [Byre] Added alignment support, and set to 128 by default
* [Compiler/Util] Added util to merged to models (through funcs)
* [Cutlass] Upgraded to c4f6b8c6bc94ff69048492fb34df0dfaf1983933
* [GPU] Supported bare ptr for GPU args, instread of Tensor
* [LLVM] Upgraded to 4592543a01609feb4b3c19e81a9d54743e15e329
* [Linalg] Fixed a bug in fuse multi-root
* [Runtime] Adjusted RNG utils

### CODE REVIEW

**Comment by liwenchangbdbz** [comment]
ci seems using a fixed version of LLVM. @Connor-XY can you take a look?

**Comment by Connor-XY** [comment]
We need to update the version of LLVM on CI machine each time we do the update. Now it's updated.

**Review by Connor-XY — APPROVED** [review]
LGTM

File: compiler/include/byteir/Utils/ModuleUtils.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
+//===- ModuleUtils.h ----------------------------------------------- C++---===//
+//
+// Copyright 2022 ByteDance Ltd. and/or its affiliates. All rights reserved.
```
Comment (WillemJiang) [inline compiler/include/byteir/Utils/ModuleUtils.h:3]: We may need to update the year of copyright, it should be 2023.

**Pull request merged**